docs/source/tutorials/first_finetune_tutorial.rst

```diff
@@ -38,118 +44,120 @@ Then, it's as simple as:
-These can be thought of as end-to-end pipelines for training and optionally evaluating LLMs.
+These can be thought of as **singularly-focused scripts for interacting with LLMs**, including training
+inference, evaluation, and quantization.
 
 Each recipe consists of three components:
 
-* **Configurable parameters**, specified through yaml configs, command-line overrides and dataclasses
+* **Configurable parameters**, specified through yaml configs and command-line overrides
+* **Recipe script**, entry-point which puts everything together including parsing and validating configs, setting up the environment, and correctly using the recipe class
 * **Recipe class**, core logic needed for training, exposed to users through a set of APIs
-* **Recipe script**, puts everything together including parsing and validating configs, setting up the environment, and correctly using the recipe class
 
-For more information on how to select the right recipe, see the :ref:`recipe_deepdive` tutorial.
+.. note::
+
+  To learn more about the concept of "recipes", check out our technical deep-dive: :ref:`recipe_deepdive`.
+
+torchtune provides built-in recipes for finetuning on single device, on multiple devices with `FSDP <https://pytorch.org/blog/introducing-pytorch-fully-sharded-data-parallel-api/>`_,
+using memory efficient techniques like `LoRA <https://arxiv.org/abs/2106.09685>`_, and more! You can view all built-in recipes `on GitHub <https://github.com/pytorch/torchtune/tree/main/recipes>`_. You can also utilize the
+:code:`tune ls` command to print out all recipes and corresponding configs.
+
+.. code-block:: bash
+
+  $ tune ls
+  RECIPE                                   CONFIG
+  full_finetune_single_device              llama2/7B_full_low_memory
+                                           mistral/7B_full_low_memory
+  full_finetune_distributed                llama2/7B_full
+                                           llama2/13B_full
+                                           mistral/7B_full
+  lora_finetune_single_device              llama2/7B_lora_single_device
+                                           llama2/7B_qlora_single_device
+                                           mistral/7B_lora_single_device
+  ...
+
+For the purposes of this tutorial, you'll will be using the recipe for finetuning a Llama2 model using `LoRA <https://arxiv.org/abs/2106.09685>`_ on
+a single device. For a more in-depth discussion on LoRA in torchtune, you can see the complete :ref:`lora_finetune_label` tutorial.
+
+.. note::
+
+  **Why have a separate recipe for single device vs. distributed?** This is discussed in
+  :ref:`recipe_deepdive` but one of our core principles in torchtune is minimal abstraction and boilerplate code.
+  If you don't want to train in a distributed environment, you shouldn't need to wade through hundreds of lines of complicated
+  distributed-specific code in order to understand how your model is training.
```

Comment:
Imo this is worded too strongly and should be framed positively rather than negatively, e.g. "If you only want to train on a single GPU, our single-device recipe ensures you don't have to worry about additional features like FSDP that are only required for distributed training."